frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js

```diff
@@ -32,44 +68,92 @@ const ReviewSubmit = ({
       </Helmet>
       <Accordion bordered={false} items={reviewItems} />
       <Container skipTopPadding className="margin-top-2 padding-top-2">
-        <h3>Submit Report</h3>
-        {submitted
-        && (
-        <Alert noIcon className="margin-y-4" type="success">
-          <b>Success</b>
-          <br />
-          This report was successfully submitted for approval
-        </Alert>
-        )}
-        {!allComplete
-        && (
+        {error && (
         <Alert noIcon className="margin-y-4" type="error">
-          <b>Incomplete report</b>
+          <b>Error</b>
           <br />
-          This report cannot be submitted until all sections are complete
+          {error}
         </Alert>
         )}
-        <Form className="smart-hub--form-large" onSubmit={handleSubmit(onFormSubmit)}>
-          <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend="Additional Notes">
-            <Label htmlFor="additionalNotes">Creator notes</Label>
-            <Textarea inputRef={register} id="additionalNotes" name="additionalNotes" />
-          </Fieldset>
-          <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend="Review and submit report">
-            <p className="margin-top-4">
-              Submitting this form for approval means that you will no longer be in draft mode.
-              Please review all information in each section before submitting to your manager for
-              approval.
-            </p>
-            <Label htmlFor="approvingManagerId">Approving manager</Label>
-            <Dropdown id="approvingManagerId" name="approvingManagerId" inputRef={register({ setValueAs: setValue, required: true })}>
-              <option name="default" value="" disabled hidden>Select a Manager...</option>
-              {approvers.map((approver) => (
-                <option key={approver.id} value={approver.id}>{approver.name}</option>
-              ))}
-            </Dropdown>
-          </Fieldset>
-          <Button type="submit" disabled={!valid}>Submit for approval</Button>
-        </Form>
+        {!approvingManager
```

Comment:
How do people feel about moving these 2 main sections into their own components? (`!approvingManager` and `approvingManager`)

Reply:
I originally split them out but decided against (for no real reason). I can split them out again if you think that will help with readability